src/level-ext.adoc

```diff
@@ -0,0 +1,119 @@
+
+==== New capability permissions
+{cheri_levels_ext_name} introduces two new capability permissions:
+[#sl_perm,reftext="SL-permission"]
+Store Level Permission (SL):: This is a `LVLBITS` wide field that allows limiting the propagation of capabilities using the following logic: capabilities with a <<section_cap_level>> less than the inverse of <<sl_perm>> will be stored with the tag cleared.
+With `LVLBITS=1`, i.e. one bit for SL and CL each, this permission functions as follows:
+// This awkward wording above allows a multi-bit scheme where capabilities as low as level = ~store_level can be stored.
+// The one bit scheme simplifies to:
+// 1 -> allow storing as low as level ~1 = 0 -> level 0 (local) and 1 (global) is okay
+// 0 -> allow storing as low as level ~0 = 1 -> only level 1 (global)
+- If this field (as well as <<c_perm>> and <<w_perm>>) is set to 1 then capabilities may be stored via this capability regardless of their associated <<section_cap_level>>.
+- If this field is zero, then any capability with a <<section_cap_level>> of zero (i.e. _local_), will be stored with the tag cleared.
+
+[#el_perm,reftext="EL-permission"]
+Elevate Level Permission (EL):: If this permission is not set then any tagged capabilities loaded via this capability will have the <<el_perm>> cleared and the <<section_cap_level>> restricted to the <<section_cap_level>> of the authorizing capability.
+This permission is similar to the existing <<lm_perm>>, but instead of applying to the <<w_perm>> on the loaded capability it restricts the <<section_cap_level,CL>> field.
+
+.Encoding of architectural permissions for MXLEN=32 when {cheri_levels_ext_name} is implemented
+[#cap_perms_encoding_levels32,width="100%",options=header,cols="^2,^1,^1,^1,^1,^1,^1,^1,^1,^2,4",align="center"]
+|==============================================================================
+|Bits[4:3]| R | W | C | LM | EL | SL | X | ASR | Mode^1^ | Notes
+11+| *Quadrant 0: Non-capability data read/write*
+11+| bit[2] - write, bit[1] - reserved (0), bit[0] - read
+11+| _Reserved bits for future extensions are 0 so new permissions are not implicitly granted_
+| 0   |   |   |   |   |   |   |   |   | N/A | No permissions
+| 1   | ✔ |   |   |   |   |   |   |   | N/A | Data RO
+| 2-3   10+| reserved
+| 4   |   | ✔ |   |   |   |   |   |   | N/A | Data WO
+| 5   | ✔ | ✔ |   |   |   |   |   |   | N/A | Data RW
+| 6-7   10+| reserved
+11+| *Quadrant 1: Executable capabilities*
+11+| bit[0] - <<m_bit>> ({CAP_MODE_VALUE}-{cheri_cap_mode_name}, {INT_MODE_VALUE}-{cheri_int_mode_name})
+|Bits[4:3]| R | W | C | LM | EL | SL  | X | ASR | Mode^1^ |
+| 0-1   | ✔ | ✔ | ✔ | ✔  | ✔  | ∞   | ✔ |  ✔  | Mode^1^  | Execute + ASR (see <<infinite-cap>>)
+| 2-3   | ✔ |   | ✔ | ✔  | ✔  | ∞   |   |     | Mode^1^  | Execute + Data & Cap RO
+| 4-5   | ✔ | ✔ | ✔ | ✔  | ✔  | ∞   | ✔ |     | Mode^1^  | Execute + Data & Cap RW
```

Comment:
I guess by infinity you mean the maximum level i.e. the most global?

Reply:
Correct, I could also just use "Any" here?

Reply:
I am ok with the infinity symbol